core/Quoridor.Core.Test/Models/WallTests.cs

```diff
@@ -0,0 +1,33 @@
+using NUnit.Framework;
+using Quoridor.Core.Models;
+using System;
+
+namespace Quoridor.Core.Test
+{
+    public class WallTests
+    {
+        [Test]
+        public void Constructor_Not_Null_Test()
+        {
+            Point[] start = { new Point(1, 1), new Point(1, 2) };
+            Point[] end = { new Point(2, 1), new Point(2, 2) };
+            Wall wall = new Wall(start, end);
+            Assert.NotNull(wall);
+        }
+        [Test]
```

Comment:
I've added empty line between functions. Refactor it in other files too, but note that before first function and after last function should not be any empty lines.
```suggestion
        }

        [Test]
```